index.bs

```diff
@@ -1385,38 +1383,45 @@ Build a composed graph up to a given output operand into a computational graph a
   <summary>
     The <dfn method for=MLGraphBuilder>build(|outputs|)</dfn> method steps are:
   </summary>
-    1. Let |promise| be [=a new promise=].
+    1. If |outputs| is empty, then return [=a new promise=] [=rejected=] with a {{TypeError}}.
+    1. [=map/For each=] |name| → |operand| of |outputs|:
+        1. If |name| is empty, then return [=a new promise=] [=rejected=] with a {{TypeError}}.
+        1. If [=MLOperand/validating MLOperand=] given |operand| and [=this=] returns false, then return [=a new promise=] [=rejected=] with a {{TypeError}}.
+        1. If |operand| is in [=this=]'s [=MLGraphBuilder/graph=]'s [=computational graph/inputs=] or [=computational graph/constants=], then return [=a new promise=] [=rejected=] with a {{TypeError}}.
+    1. Let |operands| be a new empty [=/set=].
```

Comment:
May create an `inputs` set that contains all graph input operands being reached by traversing the graph from `outputs`.

Reply:
Creating an input set is done in ba9b7bb2ebe684ef675766e2a3982aecbf0f79c9

Reply:
Thanks for adding `inputs` set. Should we keep the `operands`? Because it may contains the intermediate operands that the converting step needs.

And as my first comment, should we also create an `operators` set and populate it during the graph traversal.

Reply:
Creating an `operators` set is done in 5980228c513f4cea8a6d9c7ceba7667af6465c04